akka-persistence-typed/src/main/scala/akka/persistence/typed/serialization/ReplicatedEventSourcingSerializer.scala

```diff
@@ -130,11 +139,50 @@ import scala.collection.immutable.TreeMap
     case CrdtCounterManifest        => counterFromBinary(bytes)
     case CrdtCounterUpdatedManifest => counterUpdatedFromBinary(bytes)
 
+    case PublishedEventManifest => publishedEventFromBinary(bytes)
+
     case _ =>
       throw new NotSerializableException(
         s"Unimplemented deserialization of message with manifest [$manifest] in [${getClass.getName}]")
   }
 
+  def publishedEventToProtoByteArray(impl: PublishedEventImpl): Array[Byte] = {
```

Comment:
is there a serializer test for ReplicatedEventSourcingSerializer

See ReliableDeliverySerializerSpec or other similar for inspiration, but we should probably start using the SerializationTestKit ourselves.

Reply:
👍 i'll do that